CHANGELOG.md

```diff
@@ -0,0 +1,104 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- [Codelists] Blank Codelists may be created and given a name.
+- [Codelists] Codelists may be created from a string conforming to the Codelist Schema.
+- [Codelists] Codelists contain a set of Codes.
+- [Codelists] Codelists may be output as a XSD simpleType restriction containing enumeration elements.
+- [Codelists] Codelist equality is based on whether the names and contained Codes are the same.
+- [Codelists] Codes may be created from an optional value and name.
+- [Codelists] Codes may be output as a XSD enumeration to be contained within a simpleType restriction.
+- [Codelists] Code equality is determined based on whether name and value are the same.
+
+- [Constants] A number of constants are defined, though should be deemed private to the library.
+
+- [Datasets] Datasets can represent each Activity and Organisation files (though not at the same time!).
+- [Datasets] A Dataset may be created either from a string or a lxml ElementTree.
+- [Datasets] The contents of a Dataset can be accessed as each a string and an ElementTree. These values remain in sync with each other.
+- [Datasets] The version of the IATI Standard at which a Dataset is defined can be obtained through the `version` property.
+- [Datasets] Subsections of a Dataset (indexed by line number) may be obtained in string format.
+
+- [Defaults] The `defaults` module provides access to the contents of the SSOT.
+- [Defaults] To obtain SSOT contents, the desired version of the Standard must be specified. Should a version not be specified, the latest version will be assumed.
+- [Defaults] Specific Codelists from the SSOT may be accessed by name.
+- [Defaults] All Codelists from the SSOT may be accessed at once. No differentiation is made between Embedded and Non-Embedded Codelists.
+- [Defaults] The Standard Ruleset may be accessed.
+- [Defaults] The Ruleset Schema may be accessed. *NOTE:* This is a JSON (dict) object, not an actual Schema.
```

Comment:
Maybe make clearer but specifying it's not an iati.core.Schema, rather than Schema on it's own as it is a lowercase schema so this wording may be confusing.